ffs/arbitration.md

```diff
@@ -0,0 +1,114 @@
+
+## 4. Communications
+
+Following the formation of the tribunal, all communications between the tribunal and any interested parties shall take place in the issue opened  for the dispute.
+
+## 5. Public Hearing
+
+The tribunal shall add the dispute as a meeting item for the next scheduled Monero meeting on Freenode | OFTC | Irc2P | Slack `#monero-dev`. The Contractor and any interested parties may present arguments and/or make comments, which the tribunal may consider in evaluating the dispute.
+
+## 6. Deliberation
+
+After the Monero meeting, the tribunal shall convene over an IRC private channel created for the dispute to discuss the case. A decision shall be made within 72 hours of the close of the Monero meeting.
+
+## 7. Rules Applicable to Substance of Dispute
+
+The tribunal is explicitly authorized by the parties to resolve any dispute, *ex aequo et bono*, in accordance with what is considered to be fair and equitable without regard to any applicable law.
+
+## 8. Conduct of the Arbitration
+
+The tribunal will make every effort to conduct the arbitration in a timely manner. The tribunal has the discretion to adopt procedural measures as they consider appropriate and to determine the admissibility, relevance, materiality and weight of any evidence. The tribunal shall make all of its decisions by majority vote.
+
+## 9. Publication of Decision
+
+Once a decision has been reached, a plain text document explaining the decision shall be cryptographically signed with the use of [Unique Ring Signature](https://github.com/monero-project/urs) keyring. The text of the decision and the signature hash shall be posted in the issue opened for the dispute. In the event of a split decision (2/1), arbitrators are prohibited from disclosing who was the dissenting arbitrator.
+
+## 10. Proof
+
+If a party makes a claim, it is their responsibility to support it. A screenshot is reasonable evidence if it is not manipulated. Giving details related to an event is plausible evidence if the details can be independently confirmed. Payments can be proved with the submission of Transaction IDs, Transaction Keys, Payment IDs, public address, blockchain hash and block height data. Verifying a signed message with a private key can prove ownership of a public address. Cryptographically timestamping can prove information existed at a given point in time. The tribunal may examine metadata embedded in documents and electronic communications in order to assess the evidential value of such information.
+
+## 11. Conflicts of Interest
+
+A conflict of interest arises if a party has one of the following relationships to an arbitrator:
```

Comment:
>A conflict of interest arises if a party has one of the following relationships to an arbitrator: 

If the arbitrator or member of the party is anonymous, how can any of this be truly proven?

Reply:
There is no way to be 100% certain. It kinda comes with the territory in dealing with someone over the internet. However, since the process is adversarial, if a counterparty has evidence that could give rise to [justifiable doubts](https://www.trans-lex.org/450900/_/uncitral-model-law-on-international-commercial-arbitration/#head_16) as to the impartiality or independence of an arbitrator, the arbitrator could be challenged and removed if need be.